src/spec-common/variableSubstitution.ts

```diff
@@ -82,7 +82,7 @@ function evaluateSingleVariable(replace: Replace, match: string, variable: strin
 
 	// try to separate variable arguments from variable name
 	let args: string[] = [];
-	const parts = variable.split(':');
+	const parts = variable.split(':', 3);
```

Comment:
This would prevent us from introducing additional arguments in the future because we would break existing configs.

Also: `split()` with the limit argument does not include the remaining part of the text after the limit was hit. This would need a closer look.

Reply:
If we would need additional arguments in the future _after_ the current ones it would be nice to be able to escape the `:` in the default value. Would this be an approach to go for?

something like ` "WEBSITE_URL": "${localEnv:WEBSITE_URL:https\://example.com"`